types/office-js-preview/index.d.ts

```diff
@@ -3381,16 +3383,20 @@ declare namespace Office {
              *
              * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose or Read
              *
-             * **Important**: The `options` parameter only applies to Outlook add-ins that implement the
-             * {@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-on-send-addins | on-send feature}. It was introduced in Mailbox 1.8.
+             * **Important**: The `options` parameter only applies to Outlook add-ins starting in Mailbox 1.8. Although Outlook on Android and on iOS support up to
```

Comment:
here and in office-js

```suggestion
             * **Important**: The `options` parameter was introduced in Mailbox 1.8. Although Outlook on Android and on iOS support up to
```